### DISCUSSION TITLE
Bluetooth not recognized working

### DISCUSSION THREAD

**Discussion by SONNSSS009** [SkyrilHD/Dell-E7250-Hackintosh#57] [Q&A]
Hi SkyriHD, thank you very much for sharing, I am using your EFI to drive the installation and use it, but there is one problem I can't figure out.
I need to clarify, I am using it on Dell Latitude 3350, and now the problem is that the Bluetooth is not working properly, the identified Bluetooth is BCM 4350C2, but the wireless card model in my computer is Intel Dual Band Wireless-AC 7265... Whether I replace the driver file and reboot (cold start) or whatever, it is not recognized properly, so I hope you can provide information to help me solve this problem.
thank you very much.

**Comment by SkyrilHD** [comment]
I don't know if the 4350 is supported but try adding these kexts: BrcmFirmwareData and BrcmPatchRAM3 Can be found here: https://github.com/acidanthera/BrcmPatchRAM

Make Sure the kext load order is correct

**Reply by SONNSSS009** [reply]
Hi SkyrilHD, 
Thank you for your reply, but what you have suggested I have operated, it does not seem to work, I have seen a lot of people are also encountered my situation, I will study again according to the method you said, thank you.
